### DISCUSSION TITLE
Change the color

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#4962] [Help] [answered]
Hi All,

I just wanna to change the yellow color to an other color.

So i follow the intruction : https://filamentphp.com/docs/2.x/admin/appearance#building-themes

I need to add filament.css to vite.config.js, But i can find anywhere that filement.css file.

And also I can't  find postcss.config.js file.

Anyone can help me..

Thank.
Xami

**Comment by zepfietje** [comment, marked as answer]
Create those files manually or use the Filament install command on new projects.

**Reply by altimusin** [reply]
It is possible to change the color without add an another file ?..

I mean, Can I directly update some file where have the color code there.?

**Reply by zepfietje** [reply]
Colors should be defined in the Tailwind config as explained in the docs.
